### ISSUE TITLE
Cannot import in alfred 3.8.2

### ISSUE THREAD

**Issue by smoh** [dfm/adsabs.alfredworkflow#2]
I'm using the latest of alfred version 3 (3.8.2), and it says "Workflow Incompatible. Please update to latest version of Alfred." when imported. Do you have any ideas why? Does it possibly work only with version 4?

**Comment by dfm** [comment]
Brutal! All the pieces are compatible with Alfred 3 but, even so, it looks like workflows exported from Alfred 4 (which is what I'm currently using) can't be loaded in Alfred 3. I should be able to fix that!

**Comment by dfm** [comment]
I just pushed a new release that works in Alfred 3 for me. Want to take a look?

https://github.com/dfm/adsabs.alfredworkflow/releases/tag/v0.1.2

**Comment by smoh** [comment]
It's working now. Thanks! Out of curiosity, what was the issue? The diff just looks like re-arranging a chunk of info.plist xml -- does this work both ver 3 and 4 or is there some sort of schema change?

**Comment by dfm** [comment]
Awesome. The big difference was that I made a script that exports the workflow instead of using the built in export in Alfred. This meant that I could manually edit and remove Alfred 4 specific arguments. The meat of it is here: https://github.com/dfm/adsabs.alfredworkflow/blob/master/release.py#L19-L29
Basically Alfred 4 adds a few new arguments (that I don't even use) so I had to remove them from the plist...
